### PR TITLE
Updating TP snippet to match SSG

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.9+

Issue:
N/A

Link to docs preview:
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-api.html#oadp-api-tables
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/cicd/builds/running-entitled-builds.html#builds-running-entitled-builds-with-sharedsecret-objects_running-entitled-builds
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.html#distr-tracing-config-otel-collector_deploying-distr-tracing-data-collection
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_install/distr-tracing-deploying-jaeger.html#distr-tracing-manage-es-certificates_deploying-distr-tracing-platform
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_install/distr-tracing-installing.html#distr-tracing-install-otel-operator_install-distributed-tracing
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html#distr-tracing-rn-technology-preview_distributed-tracing-release-notes
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/authentication/identity_providers/configuring-request-header-identity-provider.html#sspi-windows_configuring-request-header-identity-provider
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#nodes-clusters-cgroups-2_post-install-cluster-tasks
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/machine-configuration-tasks.html#nodes-nodes-kernel-arguments_post-install-machine-configuration-tasks
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/networking/metallb/about-advertising-ipaddresspool.html#nw-metallb-l2padvertisement-cr_about-advertising-ip-address-pool
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-reference-smcp.html#ossm-cr-techPreview_ossm-reference
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#ossm-rn-tech-preview_ossm-release-notes
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-local.html#local-storage-discovery_persistent-storage-local
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/applications/connecting_applications_to_services/sbo-release-notes.html#sbo-release-notes-1-0-1_servicebinding-release-notes
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/applications/connecting_applications_to_services/sbo-release-notes.html#sbo-release-notes-1-0_servicebinding-release-notes
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/storage/understanding-persistent-storage.html#block-volume-support_understanding-persistent-storage
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/nodes/containers/nodes-containers-sysctls.html#updating-interface-specific-safe-sysctls-list_nodes-containers-using
* https://54469--docspreview.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-multi-network-policy.html
*  Example of a section that uses the updated TP snippet file: https://54469--docspreview.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#hosted-control-planes-overview_control-plane

QE review:
N/A

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@kalexand-rh This PR updates our TP snippet to match the [one in the SSG](https://redhat-documentation.github.io/supplementary-style-guide/#technology-preview-guidance). The only difference is that it no longer shows a bare URL when linking to the support scope article.

In doing that, I also went through and updated as many existing places that didn't use the snippet file to use it, or otherwise updated for consistency as much as possible (for ones that had extra text with it).

Let me know if you're okay with this update. If you are:
- Do you want me to wait til after GA to merge?
- Do you want me to notify the team?
- Are you good with me backporting the changes as far as possible, but not searching for new hard-coded TP mentions in older branches?

**Comment by kalexand-rh** [comment]
Thank you for making this update! I think that it looks good, and I'm ok with the update.
* If I wasn't concerned about overloading the peer review squad, it would be fine to put in before GA.
* I think that mentioning in Slack or on a team call is fine. It looks like a lot of the instances were from the aligned team, so the message might not meet the best audience.
* If you wind up making manual CPs for these, please do a quick find & replace to fix bare links. I don't think that deeper digging is necessary.

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.13` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.12. And any PR going into `main` must also target the latest version branch (enterprise-4.13).

If the update in your PR does NOT apply to version 4.13 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: modules/persistent-storage-local-discovery.adoc [hunk 1]
```diff
@@ -8,12 +8,8 @@
 
 The Local Storage Operator automates local storage discovery and provisioning. With this feature, you can simplify installation when dynamic provisioning is not available during deployment, such as with bare metal, VMware, or AWS store instances with attached devices.
 
-[IMPORTANT]
-====
-Automatic discovery and provisioning is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-
-For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
-====
+:FeatureName: Automatic discovery and provisioning
```
Comment (kcarmichael08) [inline modules/persistent-storage-local-discovery.adoc:11]: Might want to add "for local storage devices" to the feature name?

File: modules/storage-persistent-storage-block-volume.adoc [hunk 2]
```diff
@@ -41,15 +41,5 @@ The following table displays which volume plug-ins support block volumes.
...
-as fully supported, are included as a Technology Preview feature only. Technology
-Preview features are not supported with Red Hat production service level
-agreements (SLAs) and might not be functionally complete. Red Hat does not
-recommend using them in production. These features provide early access to
-upcoming product features, enabling customers to test functionality and provide
-feedback during the development process. For more information about the
-support scope of Red Hat Technology Preview
-features, see https://access.redhat.com/support/offerings/techpreview/.
-====
+:FeatureName: Using any of the block volumes that can be provisioned manually, but are not provided as fully supported,
```
Comment (kcarmichael08) [inline modules/storage-persistent-storage-block-volume.adoc:44]: Suggestion to make a bit less wordy?
```suggestion
:FeatureName: Using block volumes that can be provisioned manually, but are not provided as fully supported,
```

**Review by kcarmichael08 — COMMENTED** [review]
Had a couple of suggestions if the feature names are items that can be slightly changed. LGTM!

**Pull request merged**